src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -105,7 +106,7 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * Executes task in given context.
     */
   override def execute(): Unit = {
-    notifyToSubscribers(executeTask)
+    notifyToSubscribers(executeTask())
```

Comment:
OK. I see (http://docs.scala-lang.org/style/method-invocation.html). Thanks for clarification.